### PR TITLE
Branch help feature

### PR DESCRIPTION
**Pull request by a1021492980**
I've added the help features and some color themes that may be useful in changing color of printed texts.

### CODE REVIEW

File: src/main/java/textcolors/TextColor.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+     */
+    public static final String COLOR_RESET = "\u001B[0m";
+    public static final String COLOR_BLACK = "\u001B[30m";
+    public static final String COLOR_RED = "\u001B[31m";
+    public static final String COLOR_GREEN = "\u001B[32m";
+    public static final String COLOR_YELLOW = "\u001B[33m";
+    public static final String COLOR_BLUE = "\u001B[34m";
+    public static final String COLOR_PURPLE = "\u001B[35m";
+    public static final String COLOR_CYAN = "\u001B[36m";
+    public static final String COLOR_WHITE = "\u001B[37m";
```
Comment (alanlowzies) [inline src/main/java/textcolors/TextColor.java:18]: Didn't know you could do that. Very cool!

File: src/main/java/werkIt/UI.java [hunk 2]
```diff
@@ -119,4 +119,78 @@ public void printNewWorkoutCreatedMessage(Workout newWorkout) {
...
+    public void printHelpMessage() {
+        printListHelp();
+        printLine();
+        printWorkoutAddHelp();
+        printLine();
+        printWorkoutDeleteHelp();
+        printLine();
+        printWorkoutUpdateHelp();
+        printLine();
+        printExitHelp();
```
Comment (alanlowzies) [inline src/main/java/werkIt/UI.java:144]: Great abstraction!

**Review by alanlowzies — COMMENTED** [review]
No issues from me. I like the usage of colours to colour-code the different parts of the help command! Very nice :)

File: src/main/java/textcolors/TextColor.java (hunk 1 above)
Comment (a1021492980) [reply src/main/java/textcolors/TextColor.java:18]: Thank you!

File: src/main/java/werkIt/UI.java [hunk 3]
```diff
@@ -119,4 +119,79 @@ public void printNewWorkoutCreatedMessage(Workout newWorkout) {
...
+        System.out.println("\t This will update the workout with index 1 to 15 reps if exists.");
+    }
+
+    /**
+     * Prints help message for 'exit' command.
+     */
+    public void printExitHelp() {
+        System.out.println("\t To exit werkIt, please enter: ");
+        printColorText(COLOR_YELLOW, "\t exit");
+        System.out.println("\t ☹ This will exit werkIt. :-(");
```
Comment (emilysim00) [inline src/main/java/werkIt/UI.java:195]: I suggest to remove the smiley face at the back and front as the other help commands did not include such emoji. I feel it is a bit out of place.

Comment (a1021492980) [reply src/main/java/werkIt/UI.java:195]: No problems!

Comment (a1021492980) [reply src/main/java/werkIt/UI.java]: Issue addressed

**Pull request merged**